doc/Support/SNMP-Configuration-Examples.md

```diff
@@ -177,3 +177,60 @@ service snmpd restart
 9. In "Accepted community name" click "Add" to add your community string and permission
 10. In "Accept SNMP packets from these hosts" click "Add" and add your LibreNMS server IP address
 11. Validate change by clicking "Apply"
+
+## Configuring and Testing SNMPv3 (SNMPD)
+### Linux (Debian)
+
+#### Configuring snmpd.conf
+1. Go to /etc/snmp/snmpd.conf 
+2. Open the file in vi or nano /etc/snmp/snmpd.conf and add the following line to create SNMPV3 User (replace username and passwords with your own):
+  - ``
+createUser authPrivUser MD5 "authPassword" DES "privPassword"
+`` 
+3. Make sure the agent listens to all interfaces by adding the following line inside snmpd.conf:
+  - ``
+agentAddress udp:161,udp6:[::1]:161
+`` This line simply means listen to connections across all interfaces IPv4 and IPv6 respectively 
+4. Uncomment and change the following line to give read and write access to the username created above:
+  - ``
+#rwuser authPrivUser priv
```

Comment:
I don't think we should be suggesting rw here, we make use of only ro so should recommend that.